tests/data/rulebook.yml

```diff
@@ -3,7 +3,7 @@
   hosts: all
   sources:
     - name: range
-      range:
+      ange:
```

Comment:
@bzwei Is this to create a failure?

Reply:
no. this file should not be modified. I am removing it